src/UI/HTMLTooltip.js

```diff
@@ -26,14 +26,14 @@ export const defaultOptions = {
     wrapperClass: '',
     tooltipPositionClass: (top, left) => `
         .tooltip {
-            transform: translate(${left}px, ${top}px);
+            transform: translate(${left}px, ${top}px) !important;
         }
     `,
     caretPositionClass: (top, left, isAboveInput) => `
         .tooltip--email__caret {
             ${isAboveInput
-        ? `transform: translate(${left}px, ${top}px) rotate(180deg); transform-origin: 16px;`
-        : `transform: translate(${left}px, ${top}px);`
+        ? `transform: translate(${left}px, ${top}px) rotate(180deg); transform-origin: 16px !important;`
+        : `transform: translate(${left}px, ${top}px) !important;`
```

Comment:
We need to have `!important` on these now to make sure they override any styles set in the CSS file